packages/babel-preset-env/test/fixtures/debug/entry-corejs2-proposals-chrome-71/stdout.txt

```diff
@@ -9,6 +9,7 @@ Using modules transform: auto
 
 Using plugins:
   syntax-async-generators { "chrome":"71" }
+  proposal-numeric-separator { "chrome":"71" }
```

Comment:
When `corejs.proposals` is `true`, we should not add `proposal-numeric-separator` here because `corejs.proposals` should _not_ imply `shippedProposals: true`.